app/static/tags/securitygroup/securitygroupform.tag

```diff
@@ -0,0 +1,99 @@
+<form class="form-horizontal" role="form" name="securitygroupForm" id="securitygroupForm">
+    <div class="form-group" ng-class="{ 'has-error': securitygroupForm.sgname.$dirty && securitygroupForm.sgname.$invalid }">
+        <label class="col-sm-3 control-label" for="name">{{msg.name_label}}</label>
+
+        <div class="col-sm-9">
+            <input type="text" class="form-control" ng-pattern="/^[a-zA-Z][-a-zA-Z0-9]*$/" name="sgname" ng-model="securitygroup.name" ng-minlength="5" ng-maxlength="100" required id="sgname" placeholder="{{msg.name_placeholder}}">
+            <div class="help-block" ng-show="securitygroupForm.sgname.$dirty && securitygroupForm.sgname.$invalid">
+                <i class="fa fa-warning"></i> {{msg.template_name_invalid}}
+            </div>
+        </div>
+        <!-- .col-sm-9 -->
+
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error': securitygroupForm.description.$dirty && securitygroupForm.description.$invalid }">
+        <label class="col-sm-3 control-label" for="description">{{msg.description_label}}</label>
+
+        <div class="col-sm-9">
+            <input type="text" class="form-control" name="description" ng-model="securitygroup.description" ng-maxlength="1000" id="description" placeholder="{{msg.template_form_description_placeholder}}">
+            <div class="help-block" ng-show="securitygroupForm.description.$dirty && securitygroupForm.description.$invalid">
+                <i class="fa fa-warning"></i> {{msg.template_description_invalid}}
+            </div>
+        </div>
+        <!-- .col-sm-9 -->
+    </div>
+    <div class="form-group" ng-class="{ 'has-error': securitygroupForm.securitygroup_publicInAccount.$dirty && securitygroupForm.securitygroup_publicInAccount.$invalid }">
+            <label class="col-sm-3 control-label" for="securitygroup_publicInAccount">{{msg.public_in_account_label}}</label>
+            <div class="col-sm-9">
+                <input type="checkbox" name="securitygroup_publicInAccount" id="securitygroup_publicInAccount" ng-model="securitygroup.public">
+            </div>
+    </div>
+
+    <div class="form-group">
+        <label class="col-sm-3 control-label">Ports</label>
```

Comment:
The label could be 'security rule', because the table and the form created a role with a list of ports, protocol and the CIDR notation to allow connections from.